### DISCUSSION TITLE
How to add new map to the `CountryMap` plugin

### DISCUSSION THREAD

**Discussion by Reslan-Tinawi** [apache/superset#13512] [Q&A / Help]
Hello everyone,

First, I want to thank everyone working and contributing to this amazing project.

I installed `superset` with `docker-compose`. 

I want to use the the `CountryMap` chart to visualize some metrics. I looked at the documentation [here](https://superset.apache.org/docs/miscellaneous/country-map-tools), however, I couldn't follow the stpes.

> Put your geojson file in next folder : superset-frontend/src/visualizations/CountryMap/countries

There's no such folder, the `visualization` directory doesn't contain `CountryMap` folder.

> Go in file superset-frontend/src/explore/controls.jsx

Also this file doesn't have a `select_country` component.

Now, I understand that the front-end libraries / plugins live in another repository: [superset-ui](https://github.com/apache-superset/superset-ui), namely, I need to add changes to the [legacy-plugin-chart-country-map](https://github.com/apache-superset/superset-ui/tree/master/plugins/legacy-plugin-chart-country-map), but it's not very clear to me how to do this.

This most-recent example of [adding the Urguay map](https://github.com/apache-superset/superset-ui/pull/944) to the plugin.

In summary, I have a `.geojson` file to the country I want to add (which is Syria), but I'm not sure where should I start.

Would anyone please help me with the steps I need to follow to add a new country map, as the documentation seems a bit outdated, or correct me if I had missed something.

Thanks in advance :pray:
